composer.json

```diff
@@ -23,19 +23,28 @@
         }
     ],
     "require": {
-        "php": "^5.5|^7.0",
+        "php": "^7.1.3",
```

Comment:
Why exactly 7.1.3?
(I'm asking more out of curiosity)

Reply:
Its Laravel database package's minimum version :)

Reply:
Ok :smiley: